uber/models/department.py

```diff
@@ -150,6 +150,8 @@ class Department(MagModel):
     solicits_volunteers = Column(Boolean, default=True)
     is_shiftless = Column(Boolean, default=False)
     parent_id = Column(UUID, ForeignKey('department.id'), nullable=True)
+    is_exempt_setup = Column(Boolean, default=False)
```

Comment:
This might be really nitpicky, but these column names don't make too much sense to me. The department isn't exempt from setup or teardown, its volunteers are (from approval). Maybe something like `always_allow_setup` or `auto_approve_setup`...?

Reply:
How about `needs_setup_approval` and `needs_teardown_approval`? And then the default value would be set to `True`?

Reply:
Okay, I made that change, but I don't really like the boxes being checked by default. I don't know why, but for some reason I prefer booleans that default to false 🤷‍♂️

Reply:
I ended up with `is_setup_approval_exempt` and `is_teardown_approval_exempt`. I'm not going to spend too much more time worrying about this 😅 

> There are only two hard things in Computer Science: cache invalidation and naming things.
>
> -- Phil Karlton

Reply:
If we're throwing around relevant quotes, I offer this one (I can't remember the source and 30 seconds of Googling has failed me):

> The nice thing about booleans is that you're never wrong by more than 1 bit.

Reply:
I'll totally take `is_setup/teardown_approval_exempt`. Thanks for taking extra passes on the names! :heart: